frontend/src/components/applications/CancelApplicationDialog.vue

```diff
@@ -68,7 +68,7 @@ export default {
       try {
         this.isLoading = true
         const payload = {
-          statusCode: APPLICATION_STATUS_CODES.CANCELLED,
+          statusCode: this.sourceApplicationType === 'Application' ? APPLICATION_STATUS_CODES.CANCELLED_BY_SP : SUPPLEMENTARY_APPLICATION_STATUS_CODES.CANCELLED,
```

Comment:
I noticed that the String 'Application' is now used twice in the file. Might be worth a constant.

Reply:
it's updated